client/src/DbtLanguageClient.ts

```diff
@@ -60,6 +60,12 @@ export class DbtLanguageClient implements Disposable {
       synchronize: { fileEvents },
       outputChannel: outputChannelProvider.getMainLogChannel(),
       workspaceFolder: { uri: dbtProjectUri, name: dbtProjectUri.path, index: port },
+      middleware: {
+        didOpen: async (data: TextDocument, next: (data: TextDocument) => Promise<void>): Promise<void> =>
+          window.tabGroups.activeTabGroup.activeTab && data.uri.fsPath.endsWith(window.tabGroups.activeTabGroup.activeTab.label)
```

Comment:
Are you sure about this solution? What if there are multiple tab groups? If there are some related topics on GitHub?

Reply:
I read that tabs API should help in solving this issue on GitHub. I'm not sure 100%, I changed condition to support multiple groups. Now it works for the following example:
![image](https://user-images.githubusercontent.com/3650304/188151965-4a967f8d-1ac2-4f77-9aad-9fe18891bb6e.png)